### PR TITLE
Added the UI for the project section  

### PR DESCRIPTION
**Pull request by tanujvyas10**
Fixes Issue #389 
New feature looks like
thisone

### CODE REVIEW

**Comment by tanujvyas10** [comment]
@devesh-verma @vaibhavdaren Please review

**Comment by tanujvyas10** [comment]
> @tanujvyas10 Please Use destructuring everywhere (if possible)
> 
> 1. on clicking the remove option, the backdrop should not be dark.
> 2. The edit form in the project should fetch the project recent data of that project
> 3. Created at field should not be there in the form, it will be generated in the backend automatically when the project created
> 4. the cards in the project details section are not responsive
> 5. Do not use hardcoded data, use JSON data.
>    Thanks

Thanks for the review
About your query no. 1 , I guess, there is some bug because, I used the same procedure to create the popups for **edit and create** feature , they are working completely fine but for only delete's popup it's showing black background.I am on it . Will update soon

**Comment by tanujvyas10** [comment]
@Rupeshiya @devesh-verma @vaibhavdaren , All the required changes have been made Please review

**Comment by devesh-verma** [comment]
@tanujvyas10 there will be no option to create a new project

**Comment by tanujvyas10** [comment]
> @tanujvyas10 there will be no option to create a new project

sure sir, anything else i should add or remove?

File: src/user/projects/popups/create-project.js [hunk 1]
```diff
@@ -0,0 +1,179 @@
+import React, { Component } from "react";
+import { Modal, Button, Row, Col, Form } from "react-bootstrap";
+import "./popups.scss";
+
+export class CreateProject extends Component {
+  // eslint-disable-next-line
```
Comment (Rupeshiya) [inline src/user/projects/popups/create-project.js]: enable linting and fix if any linting error.

File: src/user/projects/popups/edit-project.js [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+  }
+
+  onChange(e) {
+    this.setState({ [e.target.name]: e.target.value });
+  }
+
+  onSubmit(e) {
+    e.preventDefault();
+    const EditProject = {
+      Proj_name: this.state.Proj_name,
```
Comment (Rupeshiya) [inline src/user/projects/popups/edit-project.js:30]: Please use destructuring

File: src/user/projects/popups/edit-project.js [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+      img_link: this.state.img_link,
+    };
+
+    console.log("Edited Project", EditProject);
+  }
+
+  componentWillReceiveProps(nextProps){
+   
+    this.setState({
+      Proj_name: nextProps.data.Proj_name,
```
Comment (Rupeshiya) [inline src/user/projects/popups/edit-project.js:46]: desctructuring

File: src/user/projects/popups/create-project.js (hunk 1 above)
Comment (tanujvyas10) [reply src/user/projects/popups/create-project.js]: @devesh-verma told me so remove the create-project feature so I,just removed it

File: src/user/projects/proj-info/proj-info.js [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+import Navigation from "../../dashboard/navigation/navigation";
+import { Card, Button, Badge, Col, Row } from "react-bootstrap";
+import "./proj-info.scss";
+import ArrowBackIcon from "@material-ui/icons/ArrowBack";
+import Fab from "@material-ui/core/Fab";
+import EditIcon from "@material-ui/icons/Edit";
+import proj_img from "../../../images/project.png";
+import user_img from "../../../images/userIcon.jpg";
+import { EditProject } from "../popups/edit-project";
+import CardActionArea from "@material-ui/core/CardActionArea";
```
Comment (Rupeshiya) [inline src/user/projects/proj-info/proj-info.js:16]: destructuring

File: src/user/projects/projects.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
+import React, { Component } from "react";
+import "./projects.scss";
+import Navigation from "../dashboard/navigation/navigation";
+import Project_list from "../../jsonData/projects";
+import { makeStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import Card from "@material-ui/core/Card";
```
Comment (Rupeshiya) [inline src/user/projects/projects.js:7]: destructuring

**Comment by tanujvyas10** [comment]
@Rupeshiya all the required changes have been and GIF has been updated.Thanks for your suggestions.Please review the latest

**Comment by vaibhavdaren** [comment]
can u resolve conflicts

**Comment by tanujvyas10** [comment]
> can u resolve conflicts

conflict has been resolved 
Please proceed

**Comment by vaibhavdaren** [comment]
conflitcs

**Comment by tanujvyas10** [comment]
> conflitcs

Conflicts have been resolved .Please proceed forward

**Pull request merged**